src/responder/common/cache_req/plugins/cache_req_common.c

```diff
@@ -26,6 +26,17 @@
 #include "providers/data_provider.h"
 #include "responder/common/cache_req/cache_req_plugin.h"
 
+errno_t cache_req_idminmax_check(struct cache_req_data *data,
+	      struct sss_domain_info *domain)
+{
+   if ((domain->id_min && (data->id < domain->id_min)) ||
+       (domain->id_max && (data->id > domain->id_max))) {
```

Comment:
Please, compare explicitly to 0 here, eg:  `domain->id_min != 0 ...`
The same comment applies for id_max.

Although it's a nitpick, it's easier to read/understand what you're checking.